Source/Common/MeshImport/NMR_MeshImporter_STL.cpp

```diff
@@ -181,6 +181,7 @@ namespace NMR {
 						VectorTree.addVector3(pNodes[j]->m_position, (nfUint32)pNodes[j]->m_index);
 					}
 				}
+				pMesh->addFace(pNodes[0], pNodes[1], pNodes[2]);
```

Comment:
I would add the faces only if they are valid.

Reply:
Line 195, 196 seems like the correct code to uncomment.

Reply:
You are absolutely right! @shahchintan14 

This is one way to get rid of compiler warnings https://github.com/3MFConsortium/lib3mf/commit/0edaeab3956ca6372d6563cde6d5700860170177#diff-2d53b9d1deb194551f0e997dd69cdc58R193 
and quite embarrassing :)